src/backoffice/dashboards/examine-management/views/section-view-examine-indexers.ts

```diff
@@ -0,0 +1,224 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { css, html, LitElement } from 'lit';
+import { customElement, state, property } from 'lit/decorators.js';
+
+import { UUIButtonState } from '@umbraco-ui/uui-button';
+
+import { UmbModalService } from '../../../../core/services/modal';
+import { UmbNotificationService } from '../../../../core/services/notification';
+import { UmbNotificationDefaultData } from '../../../../core/services/notification/layouts/default';
+import { UmbContextConsumerMixin } from '@umbraco-cms/context-api';
+import './section-view-examine-searchers';
+
+import { ApiError, ProblemDetails, Index, SearchResource } from '@umbraco-cms/backend-api';
+
+@customElement('umb-dashboard-examine-index')
+export class UmbDashboardExamineIndexElement extends UmbContextConsumerMixin(LitElement) {
+	static styles = [
+		UUITextStyles,
+		css`
+			:host {
+				display: block;
+			}
+
+			uui-box,
+			umb-dashboard-examine-searcher {
+				margin-top: var(--uui-size-space-5);
+			}
+
+			uui-box p {
+				margin-top: 0;
+			}
+
+			div.flex {
+				display: flex;
+			}
+			div.flex > uui-button {
+				padding-left: var(--uui-size-space-4);
+				height: 0;
+			}
+
+			uui-input {
+				width: 100%;
+				margin-bottom: var(--uui-size-space-5);
+			}
+
+			uui-table.info uui-table-row:first-child uui-table-cell {
+				border-top: none;
+			}
+
+			uui-table-head-cell {
+				text-transform: capitalize;
+			}
+
+			uui-table-row:last-child uui-table-cell {
+				padding-bottom: 0;
+			}
+
+			uui-icon {
+				vertical-align: top;
+				padding-right: var(--uui-size-space-5);
+			}
+
+			.positive {
+				color: var(--uui-color-positive);
+			}
+			.danger {
+				color: var(--uui-color-danger);
+			}
+
+			button {
+				background: none;
+				border: none;
+				text-decoration: underline;
+				cursor: pointer;
+			}
+			button.bright {
+				font-style: italic;
+				color: var(--uui-color-positive-emphasis);
+			}
+		`,
+	];
+
+	@property()
+	indexName!: string;
+
+	@state()
+	private _buttonState?: UUIButtonState = undefined;
+
+	@state()
+	private _indexData!: Index;
+
+	private _notificationService?: UmbNotificationService;
+	private _modalService?: UmbModalService;
+
+	private async _getIndexData() {
+		try {
+			const index = await SearchResource.getSearchIndexByIndexName({ indexName: this.indexName });
+			this._indexData = index as Index;
```

Comment:
Is this not automatically inferred as `Index` without specifying `as Index` here and elsewhere?